File: web/Models/UserModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace web.Models {
	public class User {
		public int ID {get; set;}
		public string email {get; set;}

		public string githubToken {get;set;}
		public string googleToken {get; set;}
		public string spotifyToken {get; set;}
		
	}
}

File: node/models/User.model.js
const mongoose = require("mongoose")

var UserSchema = new mongoose.Schema({
	email: {type: String, unique:true},
	githubToken: String,
	googleToken: String,
	spotifyToken: String,
	digitalOceanToken: String,
	twitchToken: String,
	password: String
})

var User = mongoose.model("user", UserSchema)

module.exports = User